### PR TITLE
Adjust acceptance tests to the latest mender-convert

### PR DESCRIPTION
**Pull request by tranchitella**
Changeelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

File: tests/test_part_image.py [hunk 1]
```diff
@@ -401,11 +401,11 @@ def test_expected_data_dirs(
                 # Expect /data/mender to exist always
                 assert "mender" in nodes
 
+                # Independently from mender-configure's installation status, we create the
+                # symlink for /var/lib/mender-configure to the data partition for mender-convert
+                # to support the subsequent installation of the add-on by the user
```
Comment (merlin-northern) [inline tests/test_part_image.py:406]: why we add this comment here?

**Review by merlin-northern — APPROVED** [review]
lgtm although I am not an expert here

File: tests/test_part_image.py (hunk 1 above)
Comment (tranchitella) [reply tests/test_part_image.py:406]: It makes sense in this context, trust me :)

**Review by merlin-northern — APPROVED** [review]
lgtm I am not an expert at least I checked that the commit matches https://github.com/mendersoftware/mender-image-tests/pull/43

**Pull request merged**